### DISCUSSION TITLE
Australian Model 3

### DISCUSSION THREAD

**Discussion by mpancino** [tesla-android/issue-tracker#200] [Q&A] [answered]
Could be very stupid question, loads well, all up and running and browser shuts on drive....drive, is there a way around this? I saw this article https://9to5mac.com/2022/06/27/tesla-carplay-workaround-all-models/ and hoped for the best. Is this only available in countries where the browser can be used in Drive?

Thanks in Advance.

**Comment by sw1002** [comment, marked as answer]
No, there is no workaround as long as You don't have root access to your car or toolbox access to set your car to a different region. However then you might get problems w/service in Your country when the car is set to another location and operated in Australia, e.g. if You set your car to be in Asia / EU and operate it in Australia to get browser in drive uncorked

**Reply by mpancino** [reply]
thanks so much, probably a good idea to update the website install guide to help people out, I had a pie lying around so no biggie, im just bummed like the rest of the southern hemisphere
